vpr/src/place/place.cpp

```diff
@@ -1405,6 +1439,18 @@ static e_move_result try_swap(const t_annealing_state* state,
         /* 1 -> move accepted, 0 -> rejected. */
         move_outcome = assess_swap(delta_c, state->t);
 
+        //Updates the manaul_move_global members and displays costs to the user to decide whether to ACCEPT/REJECT manual move.
+#ifndef NO_GRAPHICS
+        if (manual_move_flag) {
+            t_draw_state* draw_state = get_draw_state_vars();
```

Comment:
This is a long routine already so better to put all these lines in a helper function (e.g. 
move_outcome = pl_do_manual_move(...) ).